CHANGELOG.md

```diff
@@ -19,7 +24,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - Added a CloudWatch alarm to check running ElasticSearch instances, and a CloudWatch dashboard to view the health of ElasticSearch
   - Specify `AWS_REGION` in `.env` to be used by deployment script
 - **CUMULUS-670**
-  - Added new Collection file parameter "fileType" that allows configuration of workflow granule file fileType assignments
+  - Added Ancillary Metadata Export feature (see http://localhost:3000/cumulus/docs/features/ancillary_metadata for more information)
```

Comment:
I would say /docs/features.... instead of having localhost in there

Reply:
Cripes.

Reply:
Fixed.